nrf_rpc/nrf_rpc_cbor.c

```diff
@@ -190,43 +155,28 @@ void nrf_rpc_cbor_rsp_no_err(struct nrf_rpc_cbor_ctx *ctx)
 
 }
 
-void nrf_rpc_cbor_decoding_done(CborValue *value)
+void nrf_rpc_cbor_decoding_done(struct nrf_rpc_cbor_ctx *ctx)
 {
-	struct handler_proxy_ctx *ctx =
-		NRF_RPC_CONTAINER_OF(value, struct handler_proxy_ctx, value);
-
-	nrf_rpc_decoding_done(ctx->in_packet);
+	nrf_rpc_decoding_done(ctx->packet);
```

Comment:
`packet` -> `in_packet` - now it is the same (in union), but in the future someone may change the context.